storage/cloudspanner/spannerpb/spanner.proto

```diff
@@ -187,7 +165,7 @@ message TreeHead {
   reserved 5;
 
   // signature holds a digital signature across the data in this TreeHead.
-  DigitallySigned signature = 8;
+  bytes signature = 8;
```

Comment:
Is this a change that can be deployed, or does it require the cloud spanner DB to be re-created from scratch?

Reply:
This is a drop and recreate database change.  I assumed that nothing was using the cloudspanner implementation?

Reply:
+ @AlCutter 

I thought the master branch of the Trillian repo auto-deployed to a live running instance via Travis, so this would break that.

Reply:
Yes, we have a live CT/Trillian continuous deployment instance in GCE now, I'd rather we didn't break it if we can avoid it without too much headache?